static/js/components/Source.jsx

```diff
@@ -208,6 +207,74 @@ export const useSourceStyles = makeStyles((theme) => ({
   },
 }));
 
+const SourcePageThumbnails = ({
+  ra,
+  dec,
+  thumbnails,
+  rightPanelVisible,
+  downSm,
```

Comment:
downSmall, downLarge

And somewhere a comment is needed to signify what these mean

Reply:
Double checking that this cannot be handled by CSS alone

Reply:
Definitely we could, but since we are creating some of these grid and grid items with MUI + passing these size elements to the components themselves, that would require its own refactor.